docs/Actions/send-email.md

```diff
@@ -0,0 +1,531 @@
+| [Load Entities (JSON)](/docs/Actions/load-entities-json)                       | Load emails and other data to compose complex emails              |
+| [Load Entities (SQL)](/docs/Actions/load-entities-sql)                         | Load emails and other data to compose complex emails              |
+
+## `Input Parameter Reference`
+
+| Parameter                     | Description                                                                                                                                                                                                                                                               | Supports Tokens | Default                                  |
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------- | ---------------------------------------- |
+| From                          | The email address from where the email(s) will be sent                                                                                                                                                                                                                    | Yes             | `Creator of the website`                 |
+| Determine Email Automatically | If no **To** parameter is set it will try to pull the recipient email address of the logged user (current user)                                                                                                                                                           | No              | No                                       |
+| To                            | Recipient(s) to send the email to. Multiple Recipients can be separated by semicolon (;)                                                                                                                                                                                  | Yes             | `Field is required`                      |
+| Send email to all users       | Sends email to all users in the context loaded by [Load Users from SQL](/docs/Actions/load-users-sql.md)                                                                                                                                                                  | No              | No                                       |
+| Reply To                      | It is the email address that the reply message is sent when you want the reply to go to an email address that is different than the From parameter. Multiple Recipients can be separated by semicolon (;)                                                                 | Yes             | `empty string`                           |

```

Comment:
```suggestion
| Reply To                      | Email address that the reply message is sent back to instead of the `From` address. Multiple recipients can be separated by semicolon (;)                                                                 | Yes             | `empty string`                           |

```